src/commcare_cloud/ansible/setup_barman.yml

```diff
@@ -0,0 +1,5 @@
+- name: Barman
+  hosts: barman
+  become: true
+  roles:
+    - {role: barman}
```

Comment:
could we include the ssh auth setup here like we do in `deploy_db.yml` for setting up standby server auth